.github/workflows/fossa-workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: fossa-workflow
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
```

Comment:
I don't think we need all these new lines separator. (except line 2)

Reply:
thanks!.. will remove them